lib/src/styles/dimensions.dart

```diff
@@ -8,4 +8,29 @@ abstract class Dimension {
   static const double cardBorderRadius = 8.0;
   static const double clipBorderRadius = 8.0;
   static const double cardContainerEdgeInsets = 5.0;
+  static const double RatingContainerHeight = 60;
```

Comment:
These variables shoud be in camel case as well

Reply:
Solved.